.github/workflows/docker.yml

```
@@ -0,0 +1,51 @@
+name: Docker
+
+on:
+  push:
+    tags:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone Repository
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.BOT_PAT }}
+          submodules: recursive
+
+      - name: Configure Docker Metadata
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ghcr.io/emina/rosette
```

Maybe:
```suggestion
          images: ghcr.io/${{ github.repository_owner }}/rosette
```
so that forks don't fail.

---

Very good point! Or `${{ github.repository }}` altogether while we're at it. Fixed in a9c6bf6.